pages/documentation/developer/api_guide/single-line-diagram_svg-writing.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: default
+---
+
+# Single Line Diagram - SVG Writing
+
+The `com.powsybl.sld.svg.DefaultSVGWriter` class of [powsybl-single-line-diagram](../repositories/powsybl-single-line-diagram.md) is an implementation of `SVGWriter` interface which allows to generate SVG files representing single line diagrams of a substation, a voltage level or a zone. 
+We are showing in this guide how to create some single line diagrams
+- first from a test network,
+- then from a CGMES file.
```

Comment:
Is it necessary to create a list. Maybe a sentence is better:
> We are showing in this guide how to create some single line diagrams first from a test network, then from a CGMES file.

Add a link from CGMES to the format.

Reply:
Ok, changed. I linked the website page grid/formats/cim-cgmes.md, but maybe you meant the link to CGMES specifications?